common/production_methods/13_construction.txt

```diff
@@ -5,7 +5,7 @@
 
 	country_modifiers = {
 		workforce_scaled = {
-			country_construction_add = 2.5
+			country_construction_add = 2
```

Comment:
further construction nerves for low tech countries? Are we really experiencing disproportionate growth from low tech countries to necessitate this?

Reply:
Further? we reverted them for this patch. I am reverting the revert to bring it back to what it was in our previous iterations because the reasons we reverted (fears of higher construction costs for buildings like railways) ended up not being enough to stop the construction tide.

Reply:
ah yeay nevermind that. I just wanna stress that we need to account for the average (quite nooby) player, not the min-maxer. But we will see whether this is too much or too little.